lnrpc/rest-annotations.yaml

```diff
@@ -136,6 +136,9 @@ http:
     - selector: lnrpc.Lightning.ListMacaroonIDs
       post: "/v1/macaroon/list"
       body: "*"
+    - selector: lnrpc.Lightning.DeleteMacaroonID
+      post: "/v1/macaroon/delete"
```

Comment:
Should be `delete: "v1/macaroon/ids/{root_key_id}"` to be more in line with REST paradigms.

Reply:
fixed